<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperLabelTool-[Picture]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/dragon.png</normaloff>:/icons/dragon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Dir"/>
    <addaction name="actionOpen_Annotation_Dir"/>
    <addaction name="actionSet_Voc_Root"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPrev"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_RectBox"/>
    <addaction name="actionEdit_RectBox"/>
    <addaction name="actionEdit_Label"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_RectBox"/>
    <addaction name="actionHidden_RectBox"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoom"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTutorial"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar_action">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpen_Dir"/>
   <addaction name="actionOpen_Annotation_Dir"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionNext"/>
   <addaction name="actionPrev"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoom"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_RectBox"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockFilelist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>File List - [E:/]</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetImage">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>File</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QTreeWidget" name="treeWidget_file">
           <column>
            <property name="text">
             <string>File</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>OCR</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0" colspan="2">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_det">
           <property name="text">
            <string>detect</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_word">
           <property name="text">
            <string>word</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>RESIZE</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="2">
             <widget class="QRadioButton" name="radioButton_rescale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>scale</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_resize">
              <property name="text">
               <string>ON</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_refixed">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>fixed</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_3">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Pixel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>height</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>width</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2" colspan="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_rw">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="2" colspan="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_rh">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Editor</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Text</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_1">
          <property name="text">
           <string>Prev</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>AUTO OCR</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_detect">
          <property name="text">
           <string>detect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open_image.png</normaloff>:/icons/open_image.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_Dir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open_dir.png</normaloff>:/icons/open_dir.png</iconset>
   </property>
   <property name="text">
    <string>Open Image Dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionOpen_Anno">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open_anno.png</normaloff>:/icons/open_anno.png</iconset>
   </property>
   <property name="text">
    <string>Open Annotation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpen_Annotation_Dir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open_anno_dir.png</normaloff>:/icons/open_anno_dir.png</iconset>
   </property>
   <property name="text">
    <string>Open Annotation Dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save_as.png</normaloff>:/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
   </property>
   <property name="text">
    <string>Prev Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCreate_RectBox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>Create RectBox</string>
   </property>
  </action>
  <action name="actionEdit_RectBox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/edit_select.png</normaloff>:/icons/edit_select.png</iconset>
   </property>
   <property name="text">
    <string>Edit RectBox</string>
   </property>
  </action>
  <action name="actionEdit_Label">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/edit_label.png</normaloff>:/icons/edit_label.png</iconset>
   </property>
   <property name="text">
    <string>Edit Label</string>
   </property>
  </action>
  <action name="actionShow_RectBox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/show.png</normaloff>:/icons/show.png</iconset>
   </property>
   <property name="text">
    <string>Show RectBox</string>
   </property>
  </action>
  <action name="actionHidden_RectBox">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/hide.png</normaloff>:/icons/hide.png</iconset>
   </property>
   <property name="text">
    <string>Hide RectBox</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/zoom.png</normaloff>:/icons/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Original Size</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionSet_Voc_Root">
   <property name="text">
    <string>Set Pascale Voc Root</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
